### PR TITLE
Update adds-extend-domain-content.md - Adjusted readablility

### PR DESCRIPTION
**Pull request by lukemurraynz**
Update adds-extend-domain-content.md - Adjusted readability and added content related to #3836

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Please review the proposed changes. If you want to move forward with the content, add a sign-off.

#label:"aq-pr-triaged"
#label:"pending-content-team/business-approval"
Cc: @MicrosoftDocs/public-repo-pr-review-team

**Comment by Court72** [comment]
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Can you review this PR and determine whether it needs to be closed or merged?

**Comment by BryanLa** [comment]
@Court72 - LMK if you think this could be covered with a copy edit pass, or if you see anything technical that requires CACT review. @ShannonLeavitt has been helping us consolidate editorial PRs like this internally, in copy edit PRs.

**Comment by Court72** [comment]
Hi @bryanla - It looks like most of the changes could be covered with an edit pass. However, it looks like there's new content on lines 61-65 that might need CACT review.

**Comment by BryanLa** [comment]
CACT #sign-off
#remove-label:"do-not-merge"
#label:"ready-to-merge"

File: docs/example-scenario/identity/adds-extend-domain-content.md [hunk 1]
```diff
@@ -21,28 +21,28 @@ This architecture extends the hybrid network architecture shown in [Connect an o
 
 - [Azure Active Directory (Azure AD)](https://azure.microsoft.com/products/active-directory) is an enterprise identity service that provides single sign-on, multifactor authentication, and conditional access.
 - [VPN Gateway](https://azure.microsoft.com/products/vpn-gateway) is a service that uses a virtual network gateway to send encrypted traffic between an Azure virtual network and on-premises locations over the public internet. 
-- [ExpressRoute](https://azure.microsoft.com/products/expressroute) enables you to extend your on-premises networks into the Microsoft cloud over a private connection with the help of a connectivity provider.
+- [ExpressRoute](https://azure.microsoft.com/products/expressroute) enables you to extend your on-premises networks into the Microsoft Cloud over a private connection with the help of a connectivity provider.
```
Comment (Jak-MS) [inline docs/example-scenario/identity/adds-extend-domain-content.md:24]:
```suggestion
- [ExpressRoute](https://azure.microsoft.com/products/expressroute) enables you to extend your on-premises networks into the Microsoft cloud over a private connection with the help of a connectivity provider.
```

File: docs/example-scenario/identity/adds-extend-domain-content.md [hunk 2]
```diff
@@ -21,28 +21,28 @@ This architecture extends the hybrid network architecture shown in [Connect an o
...
 
-Create a separate virtual data disk for storing the database, logs, and sysvol folder for Active Directory. Don't store these items on the same disk as the operating system. By default, data disks that are attached to a VM use write-through caching. However, this form of caching can conflict with the requirements of AD DS. For this reason, set the *Host Cache Preference* setting on the data disk to *None*.
+Create a separate virtual data disk for storing the database, logs, and sysvol folder for Active Directory. Don't store these items on the same disk as the operating system. By default, data disks are attached to a VM using write-through caching. However, this caching form can conflict with AD DS's requirements. For this reason, set the *Host Cache Preference* setting on the data disk to *None*.
```
Comment (Jak-MS) [inline docs/example-scenario/identity/adds-extend-domain-content.md:45]:
```suggestion
Create a separate virtual data disk for storing the database, logs, and sysvol folder for Active Directory. Don't store these items on the same disk as the operating system. By default, data disks are attached to a VM using write-through caching. However, this form of caching can conflict with the requirements of AD DS. For this reason, set the *Host Cache Preference* setting on the data disk to *None*.
```

File: docs/example-scenario/identity/adds-extend-domain-content.md [hunk 3]
```diff
@@ -56,81 +56,87 @@ Configure the VM network interface (NIC) for each AD DS server with a static pri
...
+
+For initial configuration, you may need to adjust the Network Interface of one of your Domain Controllers in Azure, to point to a domain controller on-premises as the primary DNS source.
+
+The inclusion of its IP address in the list of DNS servers improves performance and increases the availability of DNS servers. However, if the DNS server is also a domain controller and points only to itself or points to itself first for name resolution, this can cause a delay during startup. For this reason, please be cautious when configuring the loopback address on an adapter if the server is also a domain controller.
```
Comment (Jak-MS) [inline docs/example-scenario/identity/adds-extend-domain-content.md:63]:
```suggestion
The inclusion of its IP address in the list of DNS servers improves performance and increases the availability of DNS servers. However, a start-up delay can result if the DNS server is also a domain controller and points only to itself or points to itself first for name resolution. For this reason, be cautious when configuring the loopback address on an adapter if the server is also a domain controller. 
```

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**